pkg/checks/external/nodeCheck/main.go

```diff
@@ -85,15 +85,17 @@ func waitForKuberhealthyEndpointReady(ctx context.Context, kuberhealthyEndpoint
 			return doneChan
 		default:
 		}
-
-		_, err := http.NewRequest("GET", kuberhealthyEndpoint, nil)
-		if err == nil {
+		res, err := http.Get(kuberhealthyEndpoint)
+		if err != nil {
+			log.Debugln(kuberhealthyEndpoint, "Error making reuqest to kuberhealthy endpoint "+err.Error())
```

Comment:
You've got a typo here on `reuqest` , but otherwise I think this looks sensible

Reply:
Got it, thanks!